cpp/dolfinx/fem/assemble_vector_impl.h

```diff
@@ -149,8 +119,6 @@ void _lift_bc_cells(
           {
             const T bc = bc_values1[jj];
             const T _x0 = x0.empty() ? 0.0 : x0[jj];
-            // const T _x0 = 0.0;
-            // be -= Ae.col(bs1 * j + k) * scale * (bc - _x0);
```

Comment:
Might be helpful to leave this but with an improved comment - it shows more clearly what the below mathematical operation is.

Reply:
I've left it as it was, as I don't know a nicer way of write column of Ae, except:
```c++
// Ae[:, bs1 * j + k]
```